calcom/cal.com
# refactor: Show empty panel when date not selected during adding date override

## Added panel for issue [11106](https://github.com/calcom/cal.com/issues/11106)

Related Issue
#11106 

## Requirement


## Todo
Add translations for the text "Select a date to override the placeholder"

## Type of change


## How should this be tested?



- Are there environment variables that should be set? No
- What are the minimal test data to have? No need for data
- What is expected (happy path) to have (input and output)? The panel should show when try override from route /availability

## Mandatory Tasks
Open /availability and try to add a date override.
